Wrapping/Python/CMakeLists.txt

```diff
@@ -63,113 +64,15 @@ endif()
-    set( SimpleITK_DOC_FILES_AS_LIST "${SimpleITK_DOC_FILES_AS_LIST},r${_q}${d}${_q}")
-  endforeach()
-  set( SimpleITK_DOC_FILES_AS_LIST "${SimpleITK_DOC_FILES_AS_LIST}]")
-
-endif()
-
-include( sitkConfigureFileBuildtime )
-configure_file_buildtime( "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/setup.py.in"
-  "${CMAKE_CURRENT_BINARY_DIR}/Packaging/setup.py" )
-
-configure_file(
-  "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/setupegg.py"
-  "${CMAKE_CURRENT_BINARY_DIR}/Packaging/setupegg.py"
-  COPYONLY )
-
-configure_file(
-  "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/__init__.py"
-  "${CMAKE_CURRENT_BINARY_DIR}/__init__.py"
-  COPYONLY )
-
-option(SITK_PYTHON_USE_VIRTUALENV "Create a Python Virtual Environment for testing." ON)
-mark_as_advanced(SITK_PYTHON_USE_VIRTUALENV)
-
-if (SITK_PYTHON_USE_VIRTUALENV)
-
-  # Executable to setup a new Python virtual environment
-  find_package( PythonVirtualEnv REQUIRED )
-
-  sitk_enforce_forbid_downloads( SITK_PYTHON_USE_VIRTUALENV )
-  #
-  # Setup Python Virtual Enviroment for testing and packaging
-  #
-  set( PythonVirtualenvHome "${${CMAKE_PROJECT_NAME}_BINARY_DIR}/Testing/Installation/PythonVirtualenv" )
-
-  # virtualenv places the python executable in different
-  # locations. Also note than on windows installations where python is
-  # installed only for a single user the may be a missing dll issue.
-  if( WIN32 )
-    set( VIRTUAL_PYTHON_EXECUTABLE
-      "${PythonVirtualenvHome}/Scripts/python")
-  else( )
-    set( VIRTUAL_PYTHON_EXECUTABLE "${PythonVirtualenvHome}/bin/python" )
-  endif()
-  set(TEST_PYTHON_EXECUTABLE "${VIRTUAL_PYTHON_EXECUTABLE}"
-    CACHE INTERNAL "Python executable for testing." FORCE )
-
-  # configure a scripts which creates the virtualenv and installs numpy
-  configure_file(
-    "${CMAKE_CURRENT_SOURCE_DIR}/PythonVirtualEnvInstall.cmake.in"
-    "${CMAKE_CURRENT_BINARY_DIR}/PythonVirtualEnvInstall.cmake"
-    @ONLY )
-
-  set( PythonVirtualEnv_ALL "" )
-  if ( BUILD_TESTING )
-    set( PythonVirtualEnv_ALL "ALL" )
-  endif()
-
-  add_custom_target( PythonVirtualEnv ${PythonVirtualEnv_ALL}
-    DEPENDS "${VIRTUAL_PYTHON_EXECUTABLE}"
-    SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/PythonVirtualEnvInstall.cmake.in )
-
-  add_custom_command( OUTPUT "${VIRTUAL_PYTHON_EXECUTABLE}"
-    COMMAND ${CMAKE_COMMAND} -P "${CMAKE_CURRENT_BINARY_DIR}/PythonVirtualEnvInstall.cmake"
-    DEPENDS
-    "${SWIG_MODULE_SimpleITKPython_TARGET_NAME}"
-    "${CMAKE_CURRENT_BINARY_DIR}/PythonVirtualEnvInstall.cmake"
-    ConfigureFileBuildtime
-    COMMENT "Creating python virtual enviroment..."
+install(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK.py
+    ${CMAKE_CURRENT_SOURCE_DIR}/Packaging/__init__.py
+  DESTINATION SimpleITK
+  COMPONENT Runtime
+  )
+
+install(TARGETS ${SWIG_MODULE_SimpleITKPython_TARGET_NAME}
+    RUNTIME DESTINATION SimpleITK
+    LIBRARY DESTINATION SimpleITK
+    COMPONENT Runtime
     )
-else()
-  set(TEST_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}"
```

Comment:
This variable should be need to run the testing with SimpleITK Python in the "build" tree, and not installed in the virtualenv python.